macros/multiple_databases/timestamp_from_date.sql

```diff
@@ -32,6 +32,8 @@
                 3
             )
     end
+{%- elif target.type == 'databricks' -%}
+    try_to_timestamp({{ date_field }})
```

Comment:
This will return a datetime, including the date component. We result should be a datetime, where only the time component is used, so we can use the field in calculations. We should built a timestamp out of separate components, maybe using the `extract` function?

Reply:
I think the name of the macro is a bit confusing, because it actually truncates a timestamp, but I applied the changes such that it is consistent with SQL Server and Snowflake.